SasquatchMac/SasquatchMac/ServiceStateStore.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation;
+
+@objc class ServiceStateStore : NSObject {
+  private static let AnalyticsKey : String = "MSAnalytics";
```

Comment:
Let's do this. Instead of using other keys, can you read NSUserDefaults for the keys that SDK is using?
kMSAnalyticsIsEnabledKey is an example of Analytics.